src/app/zwiazek/kalendarz/_components/Dates/Dates.tsx

```diff
@@ -0,0 +1,4 @@
+export const addMonth = (date: Date, offset = 1) =>
+  new Date(date.setMonth(date.getMonth() + offset));
+
+export const daysOfWeek = ["Pon", "Wt", "Śr", "Czw", "Pt", "Sob", "Ndz"];
```

Comment:
We are using `"default"` locale in other part of application and Polish short forms for the day of the week here.
It is a mismatch. We should have it in sync if we decide to use default locale.